src/taskpane/server.js

```diff
@@ -1210,6 +1301,39 @@ function setNumberValidation(sheet, columnNumber, rowLength, allowInvalid) {
   }
 }
 
+// create text validation on set column base
+// @param: sheet - the sheet object
+// @param: columnNumber - int of the column to validate
+// @param: rowLength - int of the number of rows for range (global param)
+// @param: allowInvalid - bool to state whether to restrict any values to those in validation or not
+function setTextValidation(sheet, columnNumber, rowLength, allowInvalid) {

```

Comment:
- [ ] Can you give a use case for this validation? Or an example of a string that should not be allowed?
- [ ] also, the allowInvalid bool is not used in the code - can you correct that?

Reply:
The idea of this function is to force Excel to make the association columns be read as text. Several different bugs were caused because of that. So, not validate the cell content itself, but its format.
The allowInvalid bool is going to be used for Google Sheets.